_data/chains/eip155-5177.json

```
@@ -0,0 +1,26 @@
+{
+    "name": "TLChain Network Mainnet",
+    "chain": "TLC",
+    "icon": "tlc",
+    "rpc": [
+      "https://mainnet-rpc.tlxscan.com/"
+    ],
+    "faucets": [],
+    "nativeCurrency": {
+      "name": "TLChain Network",
+      "symbol": "TLC",
+      "decimals": 18
+    },
+    "infoURL": "https://tlchain.network/",
+    "shortName": "tlc",
+    "chainId": 5177,
+    "networkId": 5177,
+    "explorers": [
+      {
+        "name": "TLChain Explorer",
+        "url": "https://explorer.tlchain.network/",
+        "standard": "EIP3091"
```

```suggestion
        "standard": "none"
```
seems to be not compiant